mmedit/models/editors/liif/liif.py

```diff
@@ -36,12 +36,8 @@ def forward_tensor(self, inputs, data_samples=None, **kwargs):
             Tensor: result of simple forward.
         """
 
-        coord = torch.stack([
-            data_sample.metainfo['coord'] for data_sample in data_samples
-        ]).to(inputs)
-        cell = torch.stack([
-            data_sample.metainfo['cell'] for data_sample in data_samples
-        ]).to(inputs)
+        coord = torch.stack(data_samples.metainfo['coord']).to(inputs)
```

Comment:
why do we need to stack the data samples here?

Reply:
`metainfo` are packed with list, not tensor

Reply:
I see. So, `coord` is saved in `metainfo`, which is packed as `List` by `EditDataSample.stack()`. Because `liif` need to fetch the `coord` and pass it to the `forward` function, here we need to further turn the `List` to `tensor` .